src/studio/src/designer/frontend/packages/schema-editor/src/types/index.ts

```diff
@@ -9,7 +9,7 @@ export interface ISchemaState {
   navigate?: string; // used to trigger navigation in tree, the value is not used.
 }
 export interface ILanguage {
-  [key: string]: string;
+  [key: string]: string | ILanguage;
```

Comment:
What does this actually mean? 👀

Reply:
To me, it means that the type could be a tree of ILanguage.
The value can either be a string (leaf) or a (ILanguage) branch.